README.md

```diff
@@ -1900,26 +1900,26 @@ val a: Unit < Sync =
 
 // Recurring task with a delay between
 // executions
-val b: Fiber[Unit, Any] < Sync =
+val b: Fiber[Unit, Abort[Nothing]] < Sync =
```

Comment:
I think It'd be important to not track `Abort[Nothing]` here. Sorry, I think I indicated that it'd be fine to remove `Reducible` from `Fiber.init` but I thought this would still be `Any` since the underlying impl is `IOPromiseBase[Any, A < (Async & S)]` but I'm not sure it's possible.

Reply:
I think it may still be possible to get rid of `Reducible` without tracking `Abort[Nothing]`. I'll give it a try